### DISCUSSION TITLE
SortAndBind with resorter

### DISCUSSION THREAD

**Discussion by aguahombre** [reactivemarbles/DynamicData#957] [Q&A] [answered]
Is there a way to use SortAndBind with a resorter as per the obsolete Sort method?

**Comment by JakenVeina** [comment, marked as answer]
The following overload would do the trick, yeah?

```cs
public static IObservable<IChangeSet<TObject, TKey>> SortAndBind<TObject, TKey>(
        this    IObservable<IChangeSet<TObject, TKey>>  source,
                IList<TObject>                          targetList,
                IObservable<IComparer<TObject>>         comparerChanged)
    where TObject : notnull
    where TKey : notnull;

**Reply by aguahombre** [reply]
Yes, I came to that solution as well.
Just, update the observable with the same comparer when you need to force a resort.
It works, as long as the SortAndBind method doesn't use Distinct on the observable.
